pb/lmsysmd/dataset/v1/dataset.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package lmsysmd.dataset.v1;
+
+import "buf/validate/validate.proto";
+import "google/type/date.proto";
+
+service DatasetService {
+  rpc BatchCreateDataset(BatchCreateDatasetRequest) returns (BatchCreateDatasetResponse) {
+    // Multiple calls to this endpoint will result in subsequent calls acting as no-ops.
+    option idempotency_level = IDEMPOTENT; 
+  }
+}
+
+message BatchCreateDatasetRequest {
+  string google_sheet_url = 1 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true 
+  ];
+}
+
+message BatchCreateDatasetResponse {
+  repeated Dataset datasets = 1 [ (buf.validate.field).required = true ];
+}
+
+// Each dataset points to a parent model.
+message Dataset {
+  uint32 parent = 1 [ (buf.validate.field).required = true ];
+  uint32 dataset_id = 2 [ (buf.validate.field).required = true ];
+  // The metadata consists of input type, output type, and dataset creation time.
+  map<string, string> meta = 3 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).map.min_pairs = 1,
+    (buf.validate.field).map.max_pairs = 3,
+    (buf.validate.field).map.keys = {
```

Comment:
This area looks like it has a syntax error, were you able to test the code before submitting it for review?